cmd/serve/sftp/sftp.go

```diff
@@ -110,3 +137,25 @@ sftp backend, but it may not be with other SFTP clients.
 		})
 	},
 }
+
+type StdioChannel struct {
```

Comment:
This stuff should then go into channel.go also.

Note also that it should be called `stdioChannel` with small letter to start as we don't need it exported.

Reply:
Done.